### PR TITLE
Remove unnecessary else blocks

### PR DESCRIPTION
**Pull request by Broxzier**
This PR removes unnecessary `else` bocks that are placed directly after an `if` block that ends with `return`, `continue` or `break`. This does not change the flow in any way, but cleans up the unnecessary lines and reduces the indention quite a lot in some cases.

### CODE REVIEW

File: src/openrct2-ui/WindowManager.cpp [hunk 1]
```diff
@@ -156,10 +156,7 @@ class WindowManager final : public IWindowManager
                 {
                     return this->OpenWindow(WC_RESEARCH);
                 }
-                else
-                {
-                    return window_new_ride_open_research();
-                }
+                return window_new_ride_open_research();
```
Comment (Gymnasiast) [inline src/openrct2-ui/WindowManager.cpp:159]: I prefer a space after an if block. This is, of course, a matter of preference, so feel free to decide yourself.

Comment (Broxzier) [reply src/openrct2-ui/WindowManager.cpp:159]: It's pretty mixed here. I decided to add spaces when subsequence if-statements check for different things, while keeping it as is when they check the same values. In this case it's inside a switch-case where no other cases have if-statements or empty lines, so I went without adding one.

**Comment by Gymnasiast** [comment]
Checked 40 out of 132 files so far. Looking good so far, it’s just quite a lot to check ;-)

File: src/openrct2-ui/windows/TopToolbar.cpp [hunk 2]
```diff
@@ -3041,16 +3040,14 @@ static money64 selection_lower_land(uint8_t flags)
...
-        auto landLowerAction = LandLowerAction(
-            { centreX, centreY },
-            { gMapSelectPositionA.x, gMapSelectPositionA.y, gMapSelectPositionB.x, gMapSelectPositionB.y }, gMapSelectType);
-        auto res = (flags & GAME_COMMAND_FLAG_APPLY) ? GameActions::Execute(&landLowerAction)
-                                                     : GameActions::Query(&landLowerAction);
 
-        return res->Error == GameActions::Status::Ok ? res->Cost : MONEY64_UNDEFINED;
-    }
+    auto landLowerAction = LandLowerAction(
+        { centreX, centreY }, { gMapSelectPositionA.x, gMapSelectPositionA.y, gMapSelectPositionB.x, gMapSelectPositionB.y },
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/TopToolbar.cpp:3045]: (making a note for a possible refactor here, will remove later.)

**Comment by Broxzier** [comment]
Merged using GitHub's editor, will rebase later.

File: src/openrct2/drawing/Drawing.String.cpp [hunk 3]
```diff
@@ -546,163 +548,160 @@ static void ttf_draw_string_raw_ttf(rct_drawpixelinfo* dpi, std::string_view tex
...
-        uint8_t colour = info->palette[1];
-        TTFSurface* surface = ttf_surface_cache_get_or_add(fontDesc->font, text);
-        if (surface == nullptr)
-            return;
 
-        int32_t drawX = info->x + fontDesc->offset_x;
-        int32_t drawY = info->y + fontDesc->offset_y;
-        int32_t width = surface->w;
-        int32_t height = surface->h;
+    uint8_t colour = info->palette[1];
```
Comment (Gymnasiast) [inline src/openrct2/drawing/Drawing.String.cpp:552]: colour_t (note for another PR)

File: src/openrct2/platform/Linux.cpp [hunk 4]
```diff
@@ -83,11 +83,11 @@ uint16_t platform_get_locale_language()
         {
             return LANGUAGE_ENGLISH_US;
         }
-        else if (!fnmatch(pattern, "zh_CN", 0))
+        if (!fnmatch(pattern, "zh_CN", 0))
```
Comment (Gymnasiast) [inline src/openrct2/platform/Linux.cpp:86]: (note)

File: src/openrct2/util/Util.cpp [hunk 5]
```diff
@@ -351,17 +351,14 @@ int32_t strlogicalcmp(const char* s1, const char* s2)
...
+        if (!(isdigit(*s1) && isdigit(*s2)))
         {
             if (toupper(*s1) != toupper(*s2))
                 return toupper(*s1) - toupper(*s2);
-            else
-            {
-                ++s1;
-                ++s2;
-            }
+            ++s1;
```
Comment (Gymnasiast) [inline src/openrct2/util/Util.cpp:360]: Can you put a newline above? Makes it easier to read, especially since the if does not have braces.

Comment (Broxzier) [reply src/openrct2/util/Util.cpp:360]: Sure thing. Funnily I had added one here but removed it later for consistency. I'll also add them to line 372 then.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**